public/app/features/explore/Explore.tsx

```diff
@@ -333,16 +324,12 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
     const showQueryInspector = openDrawer === ExploreDrawer.QueryInspector;
 
     return (
-      <CustomScrollbar
-        autoHeightMin={'100%'}
-        scrollRefCallback={(scrollElement) => (this.scrollElement = scrollElement || undefined)}
-      >
+      <>
         <ExploreToolbar exploreId={exploreId} onChangeTime={this.onChangeTime} />
-        {datasourceMissing ? this.renderEmptyState() : null}
-        {datasourceInstance && (
-          <div className="explore-container">
-            <div className={cx('panel-container', styles.queryContainer)}>
-              <QueryRows exploreId={exploreId} />
+
+        <div className={styles.pageContainer}>
+          <div className={styles.section}>
+            <ExploreQueries exploreId={exploreId}>
               <SecondaryActions
```

Comment:
I'm not particularly happy with passing this as a child. would be great to refactor this